scripts/simplify_network.py

```diff
@@ -114,32 +114,27 @@
 
 
 def simplify_network_to_380(n, linetype):
-    """Simplify network to v_nom == 380"""
+    """
+    Fix all lines to a voltage level of 380 kV and remove all transformers.
+    The function preserves the transmission capacity for each line while updating
+    its voltage level, line type and number of parallel bundles (num_parallel).
+    Transformers are removed and connected components are moved from their
+    starting bus to their ending bus. The corresponing starting buses are
+    removed as well.
+    """
     logger.info("Mapping all network lines onto a single 380kV layer")
 
     linetype_380 = linetype
-    lines_v_nom_b = n.lines.v_nom != 380.0
-    n.lines.loc[lines_v_nom_b, "num_parallel"] *= (
-        n.lines.loc[lines_v_nom_b, "type"].map(n.line_types.i_nom)
-        / n.line_types.loc[linetype_380, "i_nom"]
-        * n.lines.loc[lines_v_nom_b, "v_nom"]
-        / 380.0
-    )
-    n.lines.loc[lines_v_nom_b, "v_nom"] = 380.0
-    n.lines.loc[lines_v_nom_b, "type"] = linetype_380
-    n.buses["v_nom"] = 380.0
-    n.lines.loc[lines_v_nom_b, "s_nom"] = (
-        np.sqrt(3)
-        * n.line_types.loc[linetype_380, "i_nom"]
-        * n.lines.bus0.map(n.buses.v_nom)
-        * n.lines.num_parallel
-    )
+    n.lines["type"] = linetype_380
+    n.lines["v_nom"] = 380
+    n.lines["i_nom"] = n.line_types.i_nom[linetype_380]
+    n.lines["num_parallel"] = n.lines.eval("s_nom / (sqrt(3) * v_nom * i_nom)")
```

Comment:
Let's test & potentially fix this

Reply:
Tested: s_nom is defined in base_network according to line types.
I added a comment in the function to avoid future questions and notify that if an error arises there in the future we shall look at that